build_projects/dotnet-cli-build/PublishTargets.cs

```diff
@@ -550,6 +550,29 @@ private static void DownloadPackagesForPush(string pathToDownload)
             File.WriteAllText(pushedSemaphore, $"Packages pushed for build {pathToDownload}");
             AzurePublisherTool.PublishFile(pathToDownload + "/" + PackagePushedSemaphoreFileName, pushedSemaphore);
         }
+
+        [Target(nameof(PrepareTargets.Init))]
+        public static BuildTargetResult UpdateVersionsRepo(BuildTargetContext c)
+        {
+            string nupkgFilePath = EnsureVariable("NUPKG_FILE_PATH");
+            string versionsRepoPath = EnsureVariable("VERSIONS_REPO_PATH");
+
+            VersionRepoUpdater repoUpdater = new VersionRepoUpdater();
+            repoUpdater.UpdatePublishedVersions(nupkgFilePath, versionsRepoPath).Wait();
+
+            return c.Success();
+        }
+
+        private static string EnsureVariable(string variableName)
```

Comment:
This is useful, and we should probably use it all over the place, can we put it in utils?